documentation/docs/getting_started/java/android_development.md

```diff
@@ -211,12 +227,14 @@ root_app/src/main/
       libiota_client_java.so
 ```
 
-### Android studio 
+### Android studio
+
 Then using Android Studio, add the native.jar to your project by right clicking -> Add As Library... -> Select your Android app Module and press OK.
 
 ### Manual
+
 Add the jar to your `build.gradle` dependencies section using; for example: `implementation files('src\\main\\libs\\native.jar')`
 
 :::info
 When trying to build the `android-app` example in the repository with precompiled binaries, comment out all the `archTriplets` inside `iota.rs/bindings/java/examples/android-app/build.gradle`. (Otherwise you will try to regenerate the `.so` files)
```

Comment:
```suggestion
:::info

When trying to build the `android-app` example in the repository with precompiled binaries, comment out all the `archTriplets` inside `iota.rs/bindings/java/examples/android-app/build.gradle` (otherwise you will try to regenerate the `.so` files).
```